cryptopan/README.md

```diff
@@ -0,0 +1,8 @@
+<h1>CryptoPAN IP Address Anonymization</h1>
```

Comment:
Use `#` instead of `<h1>`. Also please add badges and other sections as in other crate READMEs.

Reply:
I added the badges except for crates.io, docs.rs, and build automation (since it has not been published to any of those systems)